neo/api/JSONRPC/test_json_rpc_api.py

```diff
@@ -594,3 +595,49 @@ def test_getnewaddress_with_wallet(self):
         self.app.wallet.Close()
         self.app.wallet = None
         os.remove(test_wallet_path)
+
+    def test_send_to_address_no_wallet(self):
+        req = self._gen_rpc_req("sendtoaddress", params=[])
+        mock_req = mock_request(json.dumps(req).encode("utf-8"))
+        res = json.loads(self.app.home(mock_req))
+
+        error = res.get('error', {})
+
+        self.assertEqual(error.get('code', None), -400)
+        self.assertEqual(error.get('message', None), "Access denied.")
+
+    def test_send_to_address_wrong_arguments(self):
+        test_wallet_path = os.path.join(mkdtemp(), "sendtoaddress.db3")
+        self.app.wallet = UserWallet.Create(
+            test_wallet_path,
+            to_aes_key('awesomepassword')
+        )
+
+        req = self._gen_rpc_req("sendtoaddress", params=["arg"])
+        mock_req = mock_request(json.dumps(req).encode("utf-8"))
+        res = json.loads(self.app.home(mock_req))
+
+        error = res.get('error', {})
+
+        self.assertEqual(error.get('code', None), -32602)
+        self.assertEqual(error.get('message', None), "Invalid params")
+
+        self.app.wallet.Close()
+        self.app.wallet = None
+        os.remove(test_wallet_path)
+
+    def test_send_to_address(self):
```

Comment:
I recommend replacing this test with something as shown below so the test passes like here: https://github.com/CityOfZion/neo-python/pull/588

Example:
```
    def test_send_to_address(self):
        self.app.wallet = UserWallet.Open(os.path.join(ROOT_INSTALL_PATH, "fixtures/testwallet.db3"), to_aes_key("testpassword"))
        address = 'AXjaFSP23Jkbe6Pk9pPGT6NBDs1HVdqaXK'
        neo_id = "c56f33fc6ecfcd0c225c4ab356fee59390af8560be0e930faebe74a6daff7c9b"
        req = self._gen_rpc_req("sendtoaddress", params=[neo_id, address, 1])
        mock_req = mock_request(json.dumps(req).encode("utf-8"))
        res = json.loads(self.app.home(mock_req))
        self.assertEqual(res.get('jsonrpc', None), '2.0')
        self.assertIn('txid', res.get('result', {}).keys())
        self.assertIn('vin', res.get('result', {}).keys())
        self.app.wallet.Close()
        self.app.wallet = None
```